Vikta-API-testing-sample/src/test/java/com/test/api/category/CategoryTest.java

```diff
@@ -79,4 +258,36 @@ private CategoryDTO getExistingCategoryById(int id) {
                 .extract()
                 .as(CategoryDTO.class);
     }
+
+    /**
+     * Method returns Response and expects that category exist
+     *
+     * @param id category identifier
+     * @return Response object
+     */
+    private Response getResponseFromCategoryById(int id) {
+        final String idQueryParam = "id";
+
+        return given().spec(defaultRequestSpec())
+                .when().queryParam(idQueryParam, id)
+                .get(getTestEnvironment().getCategoryPath())
+                .then()
+                .spec(defaultResponseSpec())
+                .extract()
+                .response();
+    }
+
+    /**
+     * Method used to clean after test that creates new Category
+     *
+     * @param id category identifier
+     */
+    private void cleanAfterTest(int id){
```

Comment:
try to make this method accept 'long' so You wouldn't have to downcast to 'int' every time You use it